src/components/ParticipantStrip/ParticipantStripGrid/ParticipantStripGrid.test.tsx

```diff
@@ -1,15 +1,20 @@
 import EventEmitter from 'events';
 import React from 'react';
-import ParticipantStripGrid, { ParticipantStripGridProps } from './ParticipantStripGrid';
+import ParticipantStripGrid from './ParticipantStripGrid';
 import { shallow } from 'enzyme';
 import useSelectedParticipant from '../../VideoProvider/useSelectedParticipant/useSelectedParticipant';
 import useVideoContext from '../../../hooks/useVideoContext/useVideoContext';
+import { useAppState } from '../../../state';

```

Comment:
Fix per our convention,

Reply:
https://github.com/eScribers/twilio-video-app-react/pull/72
Merge it to solve this conversation